OpenEdXMobile/src/test/java/org/edx/mobile/test/ConfigTests.java

```diff
@@ -110,35 +111,45 @@ public void testZeroRatingConfig() {
     @Test
     public void testEnrollmentNoConfig() {
         JsonObject configBase = new JsonObject();
+        JsonObject discoveryConfig = new JsonObject();
+        configBase.add(DISCOVERY, discoveryConfig);
+        JsonObject courseDiscoveryConfig = new JsonObject();
+        discoveryConfig.add(COURSE_DISCOVERY, courseDiscoveryConfig);
+
         Config config = new Config(configBase);
-        assertFalse(config.getCourseDiscoveryConfig().isCourseDiscoveryEnabled());
-        assertFalse(config.getCourseDiscoveryConfig().isWebviewCourseDiscoveryEnabled());
-        assertNull(config.getCourseDiscoveryConfig().getCourseSearchUrl());
-        assertNull(config.getCourseDiscoveryConfig().getCourseInfoUrlTemplate());
+        assertFalse(config.getDiscoveryConfig().getCourseDiscoveryConfig().isDiscoveryEnabled());
+        assertFalse(config.getDiscoveryConfig().getCourseDiscoveryConfig().isWebviewDiscoveryEnabled());
+        assertNull(config.getDiscoveryConfig().getCourseDiscoveryConfig().getBaseUrl());
+        assertNull(config.getDiscoveryConfig().getCourseDiscoveryConfig().getInfoUrlTemplate());
     }
 
     @Test
     public void testEnrollmentEmptyConfig() {
```

Comment:
`testEnrollmentNoConfig` and `testEnrollmentEmptyConfig` both methods are same, We can remove one of them.

Reply:
Good catch updated the test case.